### ISSUE TITLE
assassin creed origin game frezen when play around 30 min version 2.4  and assassin creed odyssey fps drop Bad performeance when install dxvk all version

### ISSUE THREAD

**Issue by kabutu2021** [doitsujin/dxvk#4173]
Please describe your issue as accurately as possible.

If you use Windows, please check the following page: https://github.com/doitsujin/dxvk/wiki/Windows

### Software information
Name of the game, settings used etc.

### System information
- GPU:  rtx 3090
- Driver: 555.99 -560.70
- Wine version: 
- DXVK version: 2.4

### Apitrace file(s)
- Put a link here

For instructions on how to use apitrace, see: https://github.com/doitsujin/dxvk/wiki/Using-Apitrace

### Log files
[ACOdyssey_d3d11.log](https://github.com/user-attachments/files/16405137/ACOdyssey_d3d11.log)

Please attach Proton or Wine logs as a text file:
- When using Proton, set the Steam launch options for your game to `PROTON_LOG=1 %command%` and attach the corresponding `steam-xxxxx.log` file in your home directory.
- When using regular Wine, use `wine game.exe > game.log 2>&1` and attach the resulting `game.log` file.
[ACOdyssey_dxgi.log](https://github.com/user-attachments/files/16405138/ACOdyssey_dxgi.log)

**Comment by kabutu2021** [comment]
assassin creed origin game frezen when play around 30 min version 2.4 and assassin creed odyssey fps drop Bad performeance when install dxvk all version

**Comment by Blisto91** [comment]
I've let the game sit on Windows with dxvk 2.4 for half an hour and then after that i played for 35-40 mins but i haven't reproduced anything weird. System memory and vram also seem fine.
Tested on a RTX 4080 with 555.99

**Comment by kabutu2021** [comment]
> I've let the game sit on Windows with dxvk 2.4 for half an hour and then after that i played for 35-40 mins but i haven't reproduced anything weird. System memory and vram also seem fine. Tested on a RTX 4080 with 555.99

assassin creed odyssey fps drop  and stuttering same here. Use everything normally. 

[ACOdyssey_d3d11.log](https://github.com/user-attachments/files/16440762/ACOdyssey_d3d11.log)
[ACOdyssey_dxgi.log](https://github.com/user-attachments/files/16440765/ACOdyssey_dxgi.log)

**Comment by Blisto91** [comment]
Yes I can reproduce that dxvk gives lower performance. At least with Nvidias drivers on Windows, haven't compared to Linux.
Seems to be GPU bound though

**Comment by Blisto91** [comment]
dxvk should now on current master act better when vram constrained.
